app/code/Magento/Catalog/etc/di.xml

```diff
@@ -91,12 +91,6 @@
     <type name="Magento\Framework\Mview\View\StateInterface">
         <plugin name="setStatusForMview" type="Magento\Catalog\Model\Indexer\Category\Product\Plugin\MviewState" />
     </type>
-    <type name="Magento\Catalog\Model\Indexer\Category\Product\Plugin\MviewState">
-        <arguments>
-            <argument name="state" xsi:type="object" shared="false">Magento\Framework\Mview\View\StateInterface</argument>
-            <argument name="changelog" xsi:type="object" shared="false">Magento\Framework\Mview\View\ChangelogInterface</argument>
-        </arguments>
-    </type>
     <type name="Magento\Catalog\Model\Indexer\Product\Full">
```

Comment:
why we need remove this ?

Reply:
good catch, it shouldn't, didn't seen the shared="false" at first place. I need to investigate the reason to have shared="false"